pkg/cachedinstances/cloudprovider/cached_cloud_provider_test.go

```diff
@@ -53,7 +53,7 @@ func TestCachedCloudProviderExpirationAndRefresh(t *testing.T) {
 	require.Nil(t, instance)
 	require.False(t, exists)
 	ci.IpSink() <- peekIp
-	time.Sleep(70 * time.Millisecond) // Should be refreshed couple of times and evicted.
+	time.Sleep(500 * time.Millisecond) // Should be refreshed couple of times and evicted.
```

Comment:
This test is flaky. This seem to help.

Reply:
#274 :(

Really need to get a virtual clock plumbed in to fix it properly.

Reply:
I also had trouble with that test last time I edited but I wasn't sure if increasing the time bound like that would change some property of the test. If we're sure, I have no problems with this change.

Reply:
It used to be that increasing it would change the number of items from 1 to 2 and that would fail the test - that's ultimately what #274 fixed.  If it's too low, it can go from 1 to 0 though, which is what #274 caused...

Reply:
I'm happy to revert this because it's not relevant to this PR. Especially if it's been increased and decreased before.

Reply:
Nah, it's fine, better to not be flaky.  If tests run slower, that's just motivation to add the virtual clock.